src/mapwindow.jl

```diff
@@ -41,78 +48,176 @@ and then `mapwindow(f, img, (m,n))` should filter at the 75th quantile.
 
 See also: [`imfilter`](@ref).
 """
-function mapwindow(f, img::AbstractArray, window::Dims, args...; kwargs...)
+function mapwindow(f,img,window;kw...)
+    mapwindow(f, img, window, "replicate", kw...)
```

Comment:
semicolon after `"replicate"`. Usually stylistic, but here I think it's actually a bug (like it was in https://github.com/JuliaImages/ImageView.jl/pull/130/files).

Also perhaps add spaces between the arguments?